man/validateCustomModel.Rd

```diff
@@ -0,0 +1,20 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/helper_functions.R
+\name{validateCustomModel}
+\alias{validateCustomModel}
+\title{Validate a custom caret model info list}
+\usage{
+validateCustomModel(x)
+}
+\arguments{
+\item{a}{model info list (e.g. getModelInfo("rf", regex=F)[[1]])}
```

Comment:
This miss-match is what's causing the text failure.  The argument of the function is `x` and the argument you documented is `a`.

HOWEVER, in the roxygen comment block, the documentation looks correct to me.  I think running `devtools::document()` and pushing the result will fix the problem.